coalib/results/Result.py

```diff
@@ -80,7 +85,9 @@ def __init__(self,
             raise ValueError('severity is not a valid RESULT_SEVERITY')
 
         self.origin = origin
-        self.message = message
+        self.message_base = message
+        self.message_arguments = message_arguments
+        self.message = message.format(**message_arguments)
```

Comment:
I'd rather make this a property since I could change message_base and message_arguments later